impress-org/givewp
# Feature: add 'schema-find-tour' to onboarding steps

Resolves [GIVE-535]

## Description
This PR includes this final step to the onboarding flow. Changes include:
- using the tour object to conditionally render the step from the clickExitTourCallback.
- adding the new "okay" & "nextVariant" buttons to withButtons object.
- moving inline-styles from withText object into stylesheet.
- updating the step count to not include the final "schema-find-tour" step in the withText Object.
- conditionally including the tour steps in schemaSteps.

This step of the Visual Form Builder (VFB) onboarding process within the build section aims to guide users to easily rediscover the tour. This notice will be displayed only once and is exclusively applicable to the build preview flow (schema-steps). If a user chooses to exit the tour prematurely, the tour will skip to this notice if it has not been viewed before 

## Additional context:
When the doc page for the VFB is ready an additional step for documentation will be included after the 'schema-find-tour' step.

## Affects
VFB onboarding flow for the build section

## Visuals


https://github.com/impress-org/givewp/assets/75056371/866a4bbf-d531-49e5-b5d2-93e96b70eccf


https://github.com/impress-org/givewp/assets/75056371/3d289469-1116-4703-a839-91beab1738b6


## Testing Instructions
- Remove `givewp-form-builder-schema-tour-completed` from userMeta if the flow has already been seen.
- Start the onboarding flow by visiting the build preview of a form in the VFB
- The final step should be a green modal advising users where they can find the tour again.
- You will also need to test that when selecting exit tour from a step if the find tour step has not been seen, it should take the user to that step.
- Confirm the find tour step only shows while viewing the flow the first time. Once the page has refreshed/changed it should not be seen again.
- After the the find tour step has been seen, restart the tour & test that exiting the tour works as intended. It should close the tour.

## Pre-review Checklist






[GIVE-535]: https://stellarwp.atlassian.net/browse/GIVE-535?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
